build.ps1

```diff
@@ -12,7 +12,12 @@ try {
 
     $artifactsPath = "$PSScriptRoot/artifacts"
     $packagesPath = "$artifactsPath/packages"
-    $sdkVersion = (Get-Content "$PSScriptRoot/global.json" | ConvertFrom-Json).sdk.version
+
+    $globalJson = (Get-Content "$PSScriptRoot/global.json" | ConvertFrom-Json);
```

Comment:
Gotcha here if you end up making `previousSdk` into an array - you'll want `ConvertFrom-Json -NoEnumerate`. Might want that anyway. Ran into this literally yesterday - if the array in JSON has only one member, by default `ConvertFrom-Json` will convert the array to a string. Found this because `foreach` and `.Length` started iterating over the _letters_ in a value rather than just finding one value.

Reply:
That's just generally a handy tip, thanks.